frontend/src/app/components/wp-custom-actions/wp-custom-actions/wp-custom-action.component.ts

```diff
@@ -38,6 +37,7 @@ import {HalResourceEditingService} from "core-app/modules/fields/edit/services/h
 import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 import {HalEventsService} from "core-app/modules/hal/services/hal-events.service";
 import {WorkPackageNotificationService} from "core-app/modules/work_packages/notifications/work-package-notification.service";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/e71869ec897476f98b23dceb4c3c60722bb3632a/frontend/src/app/components/wp-custom-actions/wp-custom-actions/wp-custom-action.component.ts#L41"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>